packages/solid/src/errorboundary.ts

```diff
@@ -4,8 +4,7 @@ import { mergeProps, splitProps } from 'solid-js';
 import { createComponent } from 'solid-js/web';
 
 type ErrorBoundaryProps = {
-  // eslint-disable-next-line @typescript-eslint/no-explicit-any
-  fallback: JSX.Element | ((err: any, reset: () => void) => JSX.Element);
+  fallback: JSX.Element | ((err: unknown, reset: () => void) => JSX.Element);
```

Comment:
This may also be breaking in a weird way.

Reply:
hmm true, I'll leave this as it was!